badc/source/main.rs

```diff
@@ -1,21 +1,44 @@
-use std::path::PathBuf;
+use std::{path::PathBuf, str::FromStr};
+use clap::Parser;
 
-use bad::CompilationConfiguration;
-use bad::ProgramSource;
+/// The badc compiler for the B language (Kernighan, 1969). Learning experiment for DrawsMiguel and ThePhD on Rust and some compilation techniques. Released un the CC0 1.0 Universal (e.g. Public Domain dedication).
+#[derive(Parser, Debug)]
+#[command(author, version, about = "A B language (Kernighan, 1969) compiler. Not at all useful.")]
+struct CommandLineCompilationOptions {
+	/// All of the paths to the input to compile, each one considered an independent translation unit.
+	#[arg(short, long)]
```

Comment:
Hmm, I think that for `badc foo.b` to work, you want to not set `short` or `long`, right?

Reply:
Right; I think that's how it does work. Removing!